### ISSUE TITLE
Chinese Words Separator Extension.

### ISSUE THREAD

**Issue by cunat999** [yomidevs/kaikki-to-yomitan#175]
![image](https://github.com/user-attachments/assets/a1644777-60e7-4488-a79d-1628afd73a80)

Hello, I have been using Chinese Words Separator extension, as far as I know, both are using CCEDICT, but for some reasons, Yomitan's dictionaries didn't catch "美妆" as a complete word, but "美" and "妆", which I must add separately each word into Anki, and the meaning is also not very concise.

It would be great if you can see their works. Specifically about Chinese, the extension is really good, they just don't have the Anki Connect addons.

I also want to adjust how the popups (Appearance) looks, would be great if there're guides for that.

**Comment by MarvNC** [comment]
It doesn't seem to be in [CC-CEDICT](https://www.mdbg.net/chinese/dictionary?page=worddict&wdrst=0&wdqb=%E7%BE%8E%E5%A6%86&email=). Perhaps it was in an old version of it and was later removed or something.

**Comment by cunat999** [comment]
You're right! I think it was the extension itself that do the "word-joining" therefore the word "美妆".

Thank you very much. Do you think Yomitan can do right-click on the Word and it will be expanded into smaller words/ smaller radicals? (Which I think is also beneficial for Chinese/Japanese learner, I haven't tried Japanese of Yomitan so I don't know.)

![image](https://github.com/user-attachments/assets/c2abc2c2-dd95-4133-a472-be85b15fd662)
![image](https://github.com/user-attachments/assets/de7fdf8d-a008-4c1b-ad5f-40d1b7df039d)
